package.json

```diff
@@ -1,16 +1,31 @@
 {
     "name": "backseat-debug",
     "displayName": "Backseat Debug",
-    "version": "0.4.5",
+    "version": "0.4.6",
     "publisher": "thejp",
-    "description": "Debug Extension for debugging backseat (.bs) and bssembler (.bsm) applications.",
+    "description": "Extension for the backseat language (bslang) (.bs) and bssembler (.bsm) applications:\n It provides an debugger for bssembler and syntax highlighting for bslang and bssembler",
```

Comment:
The description should be a short **one-liner**. I would suggest for example: "Backseat (bslang, .bs) and Bssembler (.bsm) – debugger for Bssembler and syntax highlighting for both."

Most Microsoft Extensions I looked at contained a dot, but some don't. So either way should be fine.